Source/WebInspectorUI/UserInterface/Views/TreeOutline.js

```diff
@@ -316,13 +316,19 @@ WI.TreeOutline = class TreeOutline extends WI.Object
         let child = this.children[childIndex];
         let parent = child.parent;
 
-        if (child.deselect(suppressOnDeselect) && !suppressSelectSibling) {
-            if (child.previousSibling)
-                child.previousSibling.select(true, false);
-            else if (child.nextSibling)
-                child.nextSibling.select(true, false);
+        let currentSelectedElement = this.treeOutline.selectedTreeElement;
+        let childOrDescentdantWasSelected = !!child.selfOrDescendant((descentdant) => descentdant === currentSelectedElement);
```

Comment:
s/`childOrDescentdantWasSelected`/`childOrDescendantWasSelected`
s/`descentdant`/`descendant`

NIT: the `!!` is kinda unnecessary since you're not using `childOrDescendantWasSelected` for anything other than a truthy check